CHANGELOG.md

```diff
@@ -64,6 +64,12 @@ When releasing, please add the new-release-boilerplate to docs/pallas/CHANGELOG.
   * Passing a DLPack capsule to {func}`jax.dlpack.from_dlpack` is deprecated.
     The argument to {func}`jax.dlpack.from_dlpack` should be an array from
     another framework that implements the ``__dlpack__`` protocol.
+  * Passing `non-array` inputs to {func}`jax.numpy.trim_zeros` is deprecated and
+    now raises a {obj}`DeprecationWarning`. It currently is converted in to an
+    array, and in the future will raise a {obj}`TypeError`. Also, passing `NdArrays`
+    with `ndim != 1` to {func}`jax.numpy.trim_zeros` is deprecated and now raises
+    a {obj}`DeprecationWarning`. It's current behavior is inconsistent with
+    {func}`numpy.trim_zeros` , and in the future will raise a {obj}`ValueError`.
```

Comment:
Changelog entries should be as concise as possible
```suggestion
  * In {func}`jax.numpy.trim_zeros`, non-arraylike arguments or arraylike arguments
    with `ndim != 1` are now deprecated, and in the future will result in an error.
```

Reply:
Thanks! Updated